src/openMVG/dataio/ImageFeed.cpp

```diff
@@ -28,15 +28,53 @@ class ImageFeed::FeederImpl
   
   FeederImpl(const std::string& imagePath, const std::string& calibPath);
   
-  bool readImage(image::Image<image::RGBColor> &imageRGB, 
+  template<typename T>
+  bool readImage(image::Image<T> &image,
```

Comment:
i don't know how it works, `readImage` is an abstract method of IFeed, does it need `override`?